pkgs/tools/networking/openapi-generator-cli/unstable.nix

```diff
@@ -1,7 +1,7 @@
 { lib, stdenv, fetchurl, jre, makeWrapper }:
 
 stdenv.mkDerivation rec {
-  version = "5.0.0-2020-02-04";
+  version = "6.0.0-2021-01-18";
```

Comment:
Can you add a comment that the fetchurl needs manual update?

Reply:
Added https://github.com/NixOS/nixpkgs/pull/110125/files#diff-99346c12886a0efd540b1fdb788464e1dd6f4719631be19f8e28b71203b3d94bR4